### ISSUE TITLE
Error rippleColor has already been defined

### ISSUE THREAD

**Issue by luminousman** [navasmdc/MaterialDesignLibrary#368]

**Comment by ropriProgrammerGitHub** [comment]
the problem is with lib google
compile 'com.google.android.gms:play-services-maps:8.4.0'

code is 
   <declare-styleable name="CustomAttributes">

        
        <attr name="rippleColor" format="color|reference" />

**Comment by easycheese** [comment]
Compile this lib instead: https://github.com/vajro/MaterialDesignLibrary

**Comment by anasanasanas123** [comment]
compile 'com.github.anasanasanas123:MaterialDesignLibrary:1.8'

**Comment by gzfgeh** [comment]
Error:(32, 13) Failed to resolve: com.github.anasanasanas123:MaterialDesignLibrary:1.8

**Comment by gzfgeh** [comment]
add compile 'com.github.anasanasanas123:MaterialDesignLibrary:1.8',then have this error:
Error:Execution failed for task ':tablelamp:dexDebug'.
> com.android.ide.common.process.ProcessException: org.gradle.process.internal.ExecException: Process 'command 'C:\Program Files\Java\jdk1.8.0_66\bin\java.exe'' finished with non-zero exit value 2

**Comment by Gordey** [comment]
How can fix it?

**Comment by anasanasanas123** [comment]
repositories {
    jcenter()
    maven { url "https://jitpack.io" }
}

dependencies {
    compile 'com.github.anasanasanas123:MaterialDesignLibrary:1.7'
}

**Comment by huylv177** [comment]
@anasanasanas123 
Failed to resolve: com.github.anasanasanas123:MaterialDesignLibrary:1.7

**Comment by anasanasanas123** [comment]
@huylv177 
did you added maven { url "https://jitpack.io" } in repositories ?

**Comment by huylv177** [comment]
@anasanasanas123 yes of course, i added it in build.gradle of whole project and the "compile..." in app module's build.gradle file.

**Comment by Horkin93** [comment]
@anasanasanas123 
Hi, i use your library and have error when use buttonRectangle.setText:
```
java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.TextView.setText(java.lang.CharSequence)' on a null object reference
                                                                              at com.gc.materialdesign.views.Button.setText(Button.java:185)
```
in this line:
```
public void setText(String text) {
        this.textButton.setText(text);
    }
```

Can you help me?

**Comment by sqro2** [comment]
this worked for me
 **in the dependencies block :**
     compile 'com.github.vajro:MaterialDesignLibrary:1.6' 

**in the repositories block :**
    maven { url "https://jitpack.io" }
    jcenter()
